test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -997,6 +997,13 @@ public static int randomNonNegativeInt() {
         return randomInt() & Integer.MAX_VALUE;
     }
 
+    /**
+     * @return an <code>int</code> between <code>Integer.MIN_VALUE</code> and <code>-1</code> (inclusive) chosen uniformly at random.
+     */
+    public static int randomNegativeInt() {
+        return randomInt() | Integer.MIN_VALUE;
```

Comment:
Nice. 

I was worried about the corner cases if randomInt returned 0 or MAX_VALUE but
`0 | Integer.MIN_VALUE == Integer.MIN_VALUE`
`Integer.MAX_VALUE | Integer.MIN_VALUE == -1`